### DISCUSSION TITLE
XKit-Rewritten on blog pages?

### DISCUSSION THREAD

**Discussion by hannahlog** [AprilSylph/XKit-Rewritten#273] [Help Desk] [answered]
Possibly dumb question: is there any way to have XKit-Rewritten run on `username.tumblr.com[/whatever]` blog pages?

After switching over from New XKit, I saw Tab Titles was not planned to be implemented, and wanted to see if I could write it as a feature myself. I created a new feature and wrote `main` to always change the document title to "Test Title"—and then realized it didn't take effect on actual blog pages, only on the dashboard, activity page, etc.

(If XKit-Rewritten is never active on blog pages, I can always write a greasemonkey script for my own personal use, just asking in case there's something I'm missing. Thanks either way!)

**Comment by invalidCards** [comment, marked as answer]
XKit Rewritten runs only on www.tumblr.com pages, so never on user blogs. I think you'll have to set up a Greasemonkey script for yourself to solve this issue.
